### ISSUE TITLE
Stikbold remapping

### ISSUE THREAD

**Issue by Shaolinmonk123** [KrossX/Durazno#24]
Hi again, wondering if you can help me out. I'm trying to remap buttons for a game called Stikbold and not having any luck. According to this tutorial the game works with x360ce which would lead me to believe it would work with Durazno?

http://steamcommunity.com/app/429330/discussions/0/350542683204957512/

I tried renaming the DLL to 1_4, kind of following what it says in the tutorial for x360ce but it doesn't work, also when I do that and then open Durazno it wont let me remap the buttons in Durazno anymore... When the DLL is 1_3 it lets me remap the buttons in Durazno but it has no effect on the game either way...

Thank you sir.

**Comment by KrossX** [comment]
Yes, the config thingy only works with the dll named 1_3. That's because the usage wasn't meant to be local, so you make the config and just copy the necessary files to the game.

As for the game itself, I have no idea. There should be a log that says if Durazno is in use. The name of the dll should match what the game is looking for.
